torch/lib/c10d/Utils.cpp

```diff
@@ -231,6 +236,23 @@ int connect(
         if (!wait || (!anyRefused && !anyReset)) {
           throw;
         }
+
+        /*
+        if a timeout is specified, check time elapsed to see if we need to
+        timeout. A timeout is specified if timeout != kNoTimeout.
+        */
```

Comment:
Comment style is `//` pretty much everywhere. There are some blocks with `/* xyz */` but those are typically limited to large chunks of documentation outside of a function. Here you can stick to `//`.